provider/datagen/src/transform/cldr/fallback/mod.rs

```diff
@@ -208,7 +208,7 @@ fn test_basic() {
         parents
             .get()
             .parents
-            .get_copied(b"zh-Hant-MO")
+            .get_copied(b"zh-Hant-MO".into())
```

Comment:
Can we make `&'static str` deref to `UnvalidatedStr` or is that not orphan-rule safe? In any case, string literal please.

Reply:
Not orphan rule safe, and also it's not really done to impl Deref primarily for getting coercions